Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -68,11 +70,17 @@ extension Lambda {
         /// For invocations from the AWS Mobile SDK, data about the client application and device.
         public let clientContext: String?
 
+        /// Context baggage.
```

Comment:
I think calling it "Context baggage" could lead to confusion.

```suggestion
        /// Baggage context.
```

Reply:
thank you, changed